components/profile_popover/profile_popover.scss

```diff
@@ -35,7 +35,6 @@
 
 .user-profile-popover {
     min-width: 200px;
-    max-width: 248px;
```

Comment:
@asaadmahmood Do we want to retrict the width of popover? I couldn't find anything in figma related this. Earlier, it was not restricted.

Reply:
@AshishDhama We should just make the width `240px` and remove the min/max.
All text should truncate if it reaches that width instead of going to a second line.